### ISSUE TITLE
headers may fail to resubscribe (wallet loses sync, requires reboot)

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-sdk#2445]
A user ran into the below. Before this, it disconnected/re-subbed correctly a bunch of times, so the below must have been some edge case. 

This causes the wallet not to be updated anymore and any new claims won't resolve. 

```
2019-09-05 12:18:07,454 INFO     torba.client.baseledger:135: lbc_mainnet: added BlockHeightEvent(height=628690, change=1) header blocks, final height 628690
2019-09-05 12:18:38,048 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv4.lbry.com:50001
2019-09-05 12:18:38,048 INFO     torba.client.basenetwork:175: Switching to SPV wallet server: spv2.lbry.com:50001
2019-09-05 12:18:38,049 INFO     torba.client.baseledger:291: Subscribing and updating accounts.
2019-09-05 12:18:40,822 INFO     torba.client.basenetwork:304: connected to spv4.lbry.com:50001
2019-09-05 12:18:52,022 WARNING  lbry.extras.daemon.exchange_rate_manager:75: There was a problem updating BTCLBC exchange rate information from lbry.com
2019-09-05 12:18:52,022 WARNING  lbry.extras.daemon.exchange_rate_manager:75: There was a problem updating USDBTC exchange rate information from lbry.com
2019-09-05 12:18:59,654 WARNING  torba.client.basenetwork:82: connection to spv2.lbry.com:50001 lost
2019-09-05 12:18:59,654 WARNING  torba.client.basenetwork:82: connection to spv2.lbry.com:50001 lost
2019-09-05 12:18:59,654 WARNING  torba.client.basenetwork:212: Wallet server unavailable, waiting for it to come back and retry.
2019-09-05 12:19:01,136 INFO     torba.client.basenetwork:304: connected to spv2.lbry.com:50001
2019-09-05 12:19:05,245 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv1.lbry.com:50001
2019-09-05 12:19:06,097 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv7.lbry.com:50001
2019-09-05 12:19:06,428 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv5.lbry.com:50001
2019-09-05 12:19:07,791 INFO     torba.client.basenetwork:304: connected to spv1.lbry.com:50001
2019-09-05 12:19:08,522 INFO     torba.client.basenetwork:304: connected to spv7.lbry.com:50001
2019-09-05 12:19:09,327 INFO     torba.client.basenetwork:304: connected to spv5.lbry.com:50001
2019-09-05 12:29:53,630 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv4.lbry.com:50001
2019-09-05 12:30:10,721 INFO     torba.client.basenetwork:304: connected to spv4.lbry.com:50001
```

**Comment by tzarebczan** [comment]
Other correct reconnections, with the above at the end: 
```2019-09-04 18:44:16,656 INFO     torba.client.basenetwork:304: connected to spv1.lbry.com:50001
2019-09-04 18:44:16,657 INFO     torba.client.basenetwork:175: Switching to SPV wallet server: spv7.lbry.com:50001
2019-09-04 18:44:16,657 INFO     torba.client.basenetwork:304: connected to spv7.lbry.com:50001
2019-09-04 18:44:16,657 INFO     torba.client.baseledger:291: Subscribing and updating accounts.
2019-09-04 18:44:16,678 INFO     torba.client.basenetwork:304: connected to spv2.lbry.com:50001
2019-09-04 18:44:19,525 INFO     lbry.stream.reflector.client:119: Sent reflector blob aa2a4cfe
2019-09-04 18:44:20,325 INFO     lbry.stream.reflector.client:119: Sent reflector blob f7582761
2019-09-04 18:44:20,326 INFO     lbry.stream.reflector.client:119: Sent reflector blob 6fa4bda0
2019-09-04 18:44:20,327 INFO     lbry.stream.reflector.client:119: Sent reflector blob a7a9ce13
2019-09-04 18:44:20,378 INFO     torba.client.basenetwork:179: Subscribed to headers: spv7.lbry.com:50001

2019-09-04 21:00:44,728 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv7.lbry.com:50001
2019-09-04 21:00:44,729 INFO     torba.client.basenetwork:175: Switching to SPV wallet server: spv4.lbry.com:50001
2019-09-04 21:00:44,729 INFO     torba.client.baseledger:291: Subscribing and updating accounts.
2019-09-04 21:00:44,840 INFO     lbry.stream.managed_stream:340: sent browser blob 236/766
2019-09-04 21:00:45,041 INFO     torba.client.basenetwork:179: Subscribed to headers: spv4.lbry.com:50001
2019-09-04 21:00:47,129 INFO     torba.client.basenetwork:304: connected to spv7.lbry.com:50001

2019-09-04 22:09:05,059 INFO     torba.client.basenetwork:304: connected to spv7.lbry.com:50001
2019-09-04 22:09:05,381 INFO     torba.client.basenetwork:304: connected to spv1.lbry.com:50001
2019-09-04 22:09:05,535 INFO     torba.client.basenetwork:304: connected to spv5.lbry.com:50001
2019-09-04 22:09:06,461 INFO     torba.client.basenetwork:304: connected to spv2.lbry.com:50001
2019-09-04 22:09:12,842 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv4.lbry.com:50001
2019-09-04 22:09:12,843 INFO     torba.client.basenetwork:175: Switching to SPV wallet server: spv2.lbry.com:50001
2019-09-04 22:09:12,843 INFO     torba.client.baseledger:291: Subscribing and updating accounts.
2019-09-04 22:09:13,093 INFO     torba.client.basenetwork:179: Subscribed to headers: spv2.lbry.com:50001
2019-09-04 22:09:15,811 INFO     torba.client.basenetwork:304: connected to spv4.lbry.com:50001
2019-09-04 22:09:40,262 INFO     torba.client.baseledger:135: lbc_mainnet: added BlockHeightEvent(height=628374, change=1) header blocks, final height 628374
2019-09-04 22:14:39,433 INFO     torba.client.baseledger:135: lbc_mainnet: added BlockHeightEvent(height=628375, change=1) header blocks, final height 628375
2019-09-04 22:14:45,662 INFO     torba.client.baseledger:135: lbc_mainnet: added BlockHeightEvent(height=628376, change=1) header blocks, final height 628376
2019-09-04 22:17:42,595 INFO     torba.client.baseledger:120: (lbc_mainnet) on_transaction: address=bbxDmFSc2cK1jFd13egJd9ZZFsARk79smc, height=0, is_verified=False, tx.id=73ad643a844f4f0df3b6b95fdf53985fa02705383883aa2bdb979045a017c1e1
2019-09-04 22:18:11,722 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv7.lbry.com:50001
2019-09-04 22:18:12,148 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv1.lbry.com:50001
2019-09-04 22:18:13,734 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv5.lbry.com:50001
2019-09-04 22:18:23,668 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv4.lbry.com:50001
2019-09-04 22:18:45,597 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv2.lbry.com:50001
2019-09-04 22:21:31,420 WARNING  lbry.extras.daemon.exchange_rate_manager:75: There was a problem updating USDBTC exchange rate information from lbry.com
2019-09-04 22:21:31,420 WARNING  lbry.extras.daemon.exchange_rate_manager:75: There was a problem updating BTCLBC exchange rate information from lbry.com
2019-09-04 22:21:31,879 WARNING  lbry.extras.daemon.exchange_rate_manager:75: There was a problem updating BTCLBC exchange rate information from Bittrex
2019-09-04 22:21:42,302 INFO     torba.client.basenetwork:304: connected to spv1.lbry.com:50001
2019-09-04 22:21:42,303 INFO     torba.client.basenetwork:175: Switching to SPV wallet server: spv1.lbry.com:50001
2019-09-04 22:21:42,303 INFO     torba.client.baseledger:291: Subscribing and updating accounts.
2019-09-04 22:21:42,341 INFO     torba.client.basenetwork:179: Subscribed to headers: spv1.lbry.com:50001
2019-09-04 22:21:42,376 INFO     torba.client.baseledger:135: lbc_mainnet: added BlockHeightEvent(height=628379, change=3) header blocks, final height 628379
2019-09-04 22:21:42,572 INFO     torba.client.basenetwork:304: connected to spv7.lbry.com:50001
2019-09-04 22:22:51,648 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv7.lbry.com:50001
2019-09-04 22:22:53,647 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv1.lbry.com:50001
2019-09-04 22:22:54,030 INFO     torba.client.basenetwork:304: connected to spv7.lbry.com:50001
2019-09-04 22:22:54,030 INFO     torba.client.basenetwork:175: Switching to SPV wallet server: spv7.lbry.com:50001
2019-09-04 22:22:54,031 INFO     torba.client.baseledger:291: Subscribing and updating accounts.
2019-09-04 22:22:54,160 INFO     torba.client.basenetwork:179: Subscribed to headers: spv7.lbry.com:50001
2019-09-04 22:22:54,315 INFO     torba.client.basenetwork:304: connected to spv2.lbry.com:50001
2019-09-04 22:22:54,695 INFO     torba.client.basenetwork:304: connected to spv1.lbry.com:50001
2019-09-04 22:22:54,784 INFO     torba.client.basenetwork:304: connected to spv4.lbry.com:50001
2019-09-04 22:22:54,916 INFO     torba.client.basenetwork:304: connected to spv5.lbry.com:50001

2019-09-04 23:34:10,510 INFO     torba.client.basenetwork:304: connected to spv1.lbry.com:50001
2019-09-04 23:34:10,511 INFO     torba.client.basenetwork:175: Switching to SPV wallet server: spv1.lbry.com:50001
2019-09-04 23:34:10,511 INFO     torba.client.baseledger:291: Subscribing and updating accounts.
2019-09-04 23:34:10,550 INFO     torba.client.basenetwork:179: Subscribed to headers: spv1.lbry.com:50001
2019-09-04 23:34:10,588 INFO     torba.client.basenetwork:304: connected to spv2.lbry.com:50001
2019-09-04 23:34:10,938 INFO     torba.client.basenetwork:304: connected to spv7.lbry.com:50001
2019-09-04 23:34:10,962 INFO     torba.client.basenetwork:304: connected to spv4.lbry.com:50001
2019-09-04 23:34:11,026 INFO     torba.client.basenetwork:304: connected to spv5.lbry.com:50001

2019-09-05 01:51:51,881 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv4.lbry.com:50001
2019-09-05 01:51:51,881 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv2.lbry.com:50001
2019-09-05 01:51:51,882 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv5.lbry.com:50001
2019-09-05 01:51:51,882 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv1.lbry.com:50001
2019-09-05 01:51:51,882 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv7.lbry.com:50001
2019-09-05 01:51:54,881 INFO     lbry.stream.reflector.client:119: Sent reflector blob 077fef60
2019-09-05 01:51:55,178 INFO     torba.client.basenetwork:304: connected to spv4.lbry.com:50001
2019-09-05 01:51:55,178 INFO     torba.client.basenetwork:304: connected to spv2.lbry.com:50001
2019-09-05 01:51:55,179 INFO     torba.client.basenetwork:304: connected to spv1.lbry.com:50001
2019-09-05 01:51:55,181 INFO     torba.client.basenetwork:175: Switching to SPV wallet server: spv4.lbry.com:50001
2019-09-05 01:51:55,185 INFO     torba.client.baseledger:291: Subscribing and updating accounts.
2019-09-05 01:51:55,328 INFO     torba.client.basenetwork:179: Subscribed to headers: spv4.lbry.com:50001
2019-09-05 01:51:55,334 INFO     torba.client.basenetwork:304: connected to spv7.lbry.com:50001
2019-09-05 01:51:55,418 INFO     torba.client.basenetwork:304: connected to spv5.lbry.com:50001

2019-09-05 12:18:07,454 INFO     torba.client.baseledger:135: lbc_mainnet: added BlockHeightEvent(height=628690, change=1) header blocks, final height 628690
2019-09-05 12:18:38,048 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv4.lbry.com:50001
2019-09-05 12:18:38,048 INFO     torba.client.basenetwork:175: Switching to SPV wallet server: spv2.lbry.com:50001
2019-09-05 12:18:38,049 INFO     torba.client.baseledger:291: Subscribing and updating accounts.
2019-09-05 12:18:40,822 INFO     torba.client.basenetwork:304: connected to spv4.lbry.com:50001
2019-09-05 12:18:52,022 WARNING  lbry.extras.daemon.exchange_rate_manager:75: There was a problem updating BTCLBC exchange rate information from lbry.com
2019-09-05 12:18:52,022 WARNING  lbry.extras.daemon.exchange_rate_manager:75: There was a problem updating USDBTC exchange rate information from lbry.com
2019-09-05 12:18:59,654 WARNING  torba.client.basenetwork:82: connection to spv2.lbry.com:50001 lost
2019-09-05 12:18:59,654 WARNING  torba.client.basenetwork:82: connection to spv2.lbry.com:50001 lost
2019-09-05 12:18:59,654 WARNING  torba.client.basenetwork:212: Wallet server unavailable, waiting for it to come back and retry.
2019-09-05 12:19:01,136 INFO     torba.client.basenetwork:304: connected to spv2.lbry.com:50001
2019-09-05 12:19:05,245 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv1.lbry.com:50001
2019-09-05 12:19:06,097 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv7.lbry.com:50001
2019-09-05 12:19:06,428 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv5.lbry.com:50001
2019-09-05 12:19:07,791 INFO     torba.client.basenetwork:304: connected to spv1.lbry.com:50001
2019-09-05 12:19:08,522 INFO     torba.client.basenetwork:304: connected to spv7.lbry.com:50001
2019-09-05 12:19:09,327 INFO     torba.client.basenetwork:304: connected to spv5.lbry.com:50001
2019-09-05 12:29:53,630 INFO     torba.client.basenetwork:89: cancelled sending server.version to spv4.lbry.com:50001
2019-09-05 12:30:10,721 INFO     torba.client.basenetwork:304: connected to spv4.lbry.com:50001

```

**Comment by tzarebczan** [comment]
Here's another log: 
```2019-09-08 10:53:51,303 INFO     torba.client.baseledger:135: lbc_mainnet: added BlockHeightEvent(height=629969, change=283) header blocks, final height 629969
2019-09-08 10:53:58,622 INFO     torba.client.basenetwork:304: connected to spv2.lbry.com:50001
2019-09-08 10:54:17,724 INFO     lbry.dht.node:158: Joined DHT, 23 peers known in 4 buckets
2019-09-08 10:55:17,730 INFO     lbry.dht.blob_announcer:42: 100 blobs to announce
2019-09-08 10:55:45,252 INFO     lbry.dht.blob_announcer:50: announced 10 blobs
2019-09-08 10:55:45,253 INFO     lbry.dht.blob_announcer:42: 90 blobs to announce
2019-09-08 10:56:01,861 INFO     torba.client.basenetwork:175: Switching to SPV wallet server: spv2.lbry.com:50001
2019-09-08 10:56:01,862 WARNING  torba.client.basenetwork:82: connection to spv5.lbry.com:50001 lost
2019-09-08 10:56:01,863 INFO     torba.client.baseledger:291: Subscribing and updating accounts.
2019-09-08 10:56:01,865 INFO     torba.client.basenetwork:86: timeout sending blockchain.block.headers to spv2.lbry.com:50001
2019-09-08 10:56:01,865 WARNING  torba.client.basenetwork:218: Wallet server call timed out, retrying.
2019-09-08 10:56:01,866 WARNING  torba.client.basenetwork:212: Wallet server unavailable, waiting for it to come back and retry.
2019-09-08 10:56:01,866 WARNING  torba.client.basenetwork:82: connection to spv2.lbry.com:50001 lost
2019-09-08 10:56:02,659 INFO     torba.client.basenetwork:304: connected to spv2.lbry.com:50001
2019-09-08 10:56:02,784 INFO     lbry.dht.blob_announcer:50: announced 10 blobs
2019-09-08 10:56:02,785 INFO     lbry.dht.blob_announcer:42: 80 blobs to announce
2019-09-08 10:56:04,427 INFO     torba.client.basenetwork:304: connected to spv5.lbry.com:50001
2019-09-08 10:56:06,767 INFO     torba.client.basenetwork:304: connected to spv5.lbry.com:50001
2019-09-08 10:56:07,090 INFO     torba.client.basenetwork:304: connected to spv2.lbry.com:50001
2019-09-08 10:56:23,106 INFO     lbry.dht.blob_announcer:50: announced 10 blobs
2019-09-08 10:56:23,107 INFO     lbry.dht.blob_announcer:42: 70 blobs to announce
2019-09-08 10:56:59,616 INFO     lbry.dht.blob_announcer:50: announced 10 blobs
2019-09-08 10:56:59,616 INFO     lbry.dht.blob_announcer:42: 60 blobs to announce
2019-09-08 10:57:06,809 WARNING  torba.client.basenetwork:82: connection to spv5.lbry.com:50001 lost
2019-09-08 10:57:07,146 WARNING  torba.client.basenetwork:82: connection to spv2.lbry.com:50001 lost
2019-09-08 10:57:09,393 INFO     torba.client.basenetwork:304: connected to spv5.lbry.com:50001
2019-09-08 10:57:09,688 INFO     torba.client.basenetwork:304: connected to spv2.lbry.com:50001
2019-09-08 10:57:24,628 INFO     lbry.dht.blob_announcer:50: announced 10 blobs
2019-09-08 10:57:24,629 INFO     lbry.dht.blob_announcer:42: 50 blobs to announce
2019-09-08 10:57:45,471 INFO     lbry.dht.blob_announcer:50: announced 10 blobs
2019-09-08 10:57:45,471 INFO     lbry.dht.blob_announcer:42: 40 blobs to announce
2019-09-08 10:57:51,558 WARNING  torba.client.basenetwork:82: connection to spv1.lbry.com:50001 lost
2019-09-08 10:57:51,653 WARNING  torba.client.basenetwork:82: connection to spv3.lbry.com:50001 lost
2019-09-08 10:57:52,237 WARNING  torba.client.basenetwork:82: connection to spv4.lbry.com:50001 lost
2019-09-08 10:57:54,560 INFO     torba.client.basenetwork:304: connected to spv1.lbry.com:50001
2019-09-08 10:57:55,264 INFO     torba.client.basenetwork:304: connected to spv3.lbry.com:50001
2019-09-08 10:57:55,829 INFO     torba.client.basenetwork:304: connected to spv4.lbry.com:50001
2019-09-08 10:58:06,679 INFO     lbry.dht.blob_announcer:50: announced 10 blobs
2019-09-08 10:58:06,680 INFO     lbry.dht.blob_announcer:42: 30 blobs to announce
2019-09-08 10:58:27,021 INFO     lbry.dht.blob_announcer:50: announced 10 blobs
2019-09-08 10:58:27,022 INFO     lbry.dht.blob_announcer:42: 20 blobs to announce
2019-09-08 10:58:52,045 INFO     lbry.dht.blob_announcer:50: announced 10 blobs
2019-09-08 10:58:52,045 INFO     lbry.dht.blob_announcer:42: 10 blobs to announce
2019-09-08 10:58:55,499 WARNING  torba.client.basenetwork:82: connection to spv5.lbry.com:50001 lost
2019-09-08 10:58:55,499 WARNING  torba.client.basenetwork:82: connection to spv3.lbry.com:50001 lost
2019-09-08 10:58:55,500 WARNING  torba.client.basenetwork:82: connection to spv4.lbry.com:50001 lost
2019-09-08 10:58:55,500 WARNING  torba.client.basenetwork:82: connection to spv1.lbry.com:50001 lost
2019-09-08 10:58:56,437 INFO     torba.client.basenetwork:304: connected to spv2.lbry.com:50001
2019-09-08 10:58:57,928 INFO     torba.client.basenetwork:304: connected to spv5.lbry.com:50001
2019-09-08 10:58:58,052 INFO     torba.client.basenetwork:304: connected to spv1.lbry.com:50001
2019-09-08 10:58:58,116 INFO     torba.client.basenetwork:304: connected to spv3.lbry.com:50001
2019-09-08 10:58:59,246 INFO     torba.client.basenetwork:304: connected to spv4.lbry.com:50001
2019-09-08 10:59:11,927 INFO     lbry.dht.blob_announcer:50: announced 10 blobs
2019-09-08 10:59:56,468 WARNING  torba.client.basenetwork:82: connection to spv2.lbry.com:50001 lost
2019-09-08 10:59:57,991 WARNING  torba.client.basenetwork:82: connection to spv5.lbry.com:50001 lost
2019-09-08 10:59:58,089 WARNING  torba.client.basenetwork:82: connection to spv1.lbry.com:50001 lost
2019-09-08 10:59:58,170 WARNING  torba.client.basenetwork:82: connection to spv3.lbry.com:50001 lost
2019-09-08 10:59:59,031 INFO     torba.client.basenetwork:304: connected to spv2.lbry.com:50001
2019-09-08 10:59:59,300 WARNING  torba.client.basenetwork:82: connection to spv4.lbry.com:50001 lost
2019-09-08 11:00:00,543 INFO     torba.client.basenetwork:304: connected to spv5.lbry.com:50001
2019-09-08 11:00:00,599 INFO     torba.client.basenetwork:304: connected to spv1.lbry.com:50001
2019-09-08 11:00:00,817 INFO     torba.client.basenetwork:304: connected to spv3.lbry.com:50001
2019-09-08 11:00:02,111 INFO     torba.client.basenetwork:304: connected to spv4.lbry.com:50001
2019-09-08 11:19:27,868 INFO     torba.client.basenetwork:304: connected to spv5.lbry.com:50001
2019-09-08 11:19:27,902 INFO     torba.client.basenetwork:304: connected to spv1.lbry.com:50001
2019-09-08 11:19:28,088 INFO     torba.client.basenetwork:304: connected to spv2.lbry.com:50001
2019-09-08 11:19:28,142 INFO     torba.client.basenetwork:304: connected to spv4.lbry.com:50001
2019-09-08 11:19:28,408 INFO     torba.client.basenetwork:304: connected to spv3.lbry.com:50001
2019-09-08 19:22:39,018 WARNING  torba.client.basenetwork:82: connection to spv4.lbry.com:50001 lost
2019-09-08 19:22:41,799 INFO     torba.client.basenetwork:304: connected to spv4.lbry.com:50001
2019-09-08 19:22:44,429 WARNING  torba.client.basenetwork:82: connection to spv1.lbry.com:50001 lost
2019-09-08 19:22:46,961 INFO     torba.client.basenetwork:304: connected to spv1.lbry.com:50001
2019-09-08 19:22:52,919 WARNING  torba.client.basenetwork:82: connection to spv5.lbry.com:50001 lost
2019-09-08 19:22:55,476 INFO     torba.client.basenetwork:304: connected to spv5.lbry.com:50001
2019-09-08 19:22:58,329 WARNING  torba.client.basenetwork:82: connection to spv2.lbry.com:50001 lost
2019-09-08 19:22:58,879 WARNING  torba.client.basenetwork:82: connection to spv3.lbry.com:50001 lost
2019-09-08 19:23:00,882 INFO     torba.client.basenetwork:304: connected to spv2.lbry.com:50001
2019-09-08 19:23:01,512 INFO     torba.client.basenetwork:304: connected to spv3.lbry.com:50001
2019-09-08 20:48:46,458 INFO     lbry.extras.daemon.Daemon:841: Get version info: {"processor": "Intel64 Family 6 Model 60 Stepping 3, GenuineIntel", "python_version": "3.7.4", "platform": "Windows-10-10.0.18362-SP0", "os_release": "10", "os_system": "Windows", "lbrynet_version": "0.40.0", "torba_version": "0.5.7", "build": "release"}
2019-09-08 20:55:50,260 INFO     lbry.extras.daemon.Daemon:587: cancelled API call for: utxo_release
2019-09-08 20:55:50,262 INFO     lbry.extras.daemon.Daemon:587: cancelled API call for: status
2019-09-08 20:55:50,263 INFO     torba.client.basenetwork:89: cancelled sending blockchain.claimtrie.resolve to spv2.lbry.com:50001
2019-09-08 20:55:50,264 INFO     lbry.extras.daemon.Daemon:587: cancelled API call for: resolve
2019-09-08 20:55:50,264 INFO     lbry.extras.daemon.Daemon:587: cancelled API call for: account_balance
2019-09-08 20:55:50,265 INFO     lbry.extras.daemon.Daemon:708: Shutting down lbrynet daemon
2019-09-08 20:55:50,265 INFO     torba.client.basenetwork:86: timeout sending server.version to spv2.lbry.com:50001
2019-09-08 20:55:50,267 INFO     lbry.stream.stream_manager:214: finished stopping the stream manager
2019-09-08 20:55:50,267 INFO     lbry.blob_exchange.server:178: Stopped blob server```

**Comment by tzarebczan** [comment]
This should be fixed by https://github.com/lbryio/lbry-sdk/pull/2451, will reopen if not.

**Comment by tzarebczan** [comment]
I ran into this with 2 servers in my config file - @shyba said he'd add more logging to see what may have happened.

**Comment by tzarebczan** [comment]
Here's a recent log from Victor's latest branch - could not run claim search afterward and it lost sync: 
https://gist.github.com/tzarebczan/cde82512738390d1d102aa4c9a44415b

**Comment by lyoshenka** [comment]
fixed in #2479
